src/style.css

```
@@ -1 +1,70 @@
 /* Styles go here */
+* {
+  /* box-sizing: border-box; */
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  margin: 0 auto;
+  /* width: 1200px; */
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.menu {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.list {
+  margin-left: 20px;
+  height: 60px;
+  display: flex;
+  align-items: center;
+}
```

```suggestion
.list {
  margin-left: 20px;
}
```
nav item will be responsible only for margins to other items